File: problems/problems_3234/Solution.java
package problems.problems_3234;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public int numberOfSubstrings(String s) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        String s = jsonStringToString(inputJsonValues[0]);
        return JSON.toJSON(numberOfSubstrings(s));
    }
}

File: problems/problems_3234/solution.go
package problem3234

import (
	"encoding/json"
	"log"
	"strings"
)

func numberOfSubstrings(s string) int {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var s string

	if err := json.Unmarshal([]byte(inputValues[0]), &s); err != nil {
		log.Fatal(err)
	}

	return numberOfSubstrings(s)
}
